src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: $card-width;
+  border: $card-border;
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    height: 134px;
+    width: 160px;
+    background-size: 160px 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    color: $color-main;
```

Comment:
The variable `$color-main` is used here, but it's not defined within this file. Ensure that it is defined in a file that is imported before this one or within this file itself.